### ISSUE TITLE
Use use_ssl option

### ISSUE THREAD

**Issue by nazrulcse** [eventmachine/eventmachine#880]
uri = URI(url)
http = Net::HTTP.new(uri.host, uri.port)
req = Net::HTTP::Post.new(uri.path, {'Content-Type' =>'application/json'})
http.use_ssl = true
http.request(req)

How to use **http.use_ssl = true** in eventmachine?

EventMachine::HttpRequest.new(url, {:ssl => {use: true}}).post :body => req_body

**Comment by sodabrew** [comment]
Do you mean for this gem? https://github.com/igrigorik/em-http-request

**Comment by nazrulcse** [comment]
Yes, but https:// is not working. Getting 'connection closed by server' response. But Net::HTTP working perfectly 
uri = URI(url)
http = Net::HTTP.new(uri.host, uri.port)
req = Net::HTTP::Post.new(uri.path, {'Content-Type' =>'application/json'})
http.use_ssl = true
http.request(req)

**Comment by sodabrew** [comment]
Net::HTTP is part of Ruby's standard library.

EventMachine::HttpRequest lives here: https://github.com/igrigorik/em-http-request

**Comment by nazrulcse** [comment]
Ok, Then how can i work this on https://github.com/igrigorik/em-http-request any suggestion.
